src/programs/menu/break-handler/addBreak.ts

```diff
@@ -0,0 +1,44 @@
+import prisma from "../../../prisma";
+import { GuildMember, DMChannel } from "discord.js";
+import { textLog } from "../../../common/moderator";
+import state from "../../../common/state";
+import Tools from "../../../common/tools";
+import { removeIgnore, logger, handleError, emojiCollector } from "../common";
+
+export const addBreakRole = async (
+  member: GuildMember,
+  dmChannel: DMChannel
+) => {
+  const confirmationMessage = await dmChannel.send(
+    "It looks like you want a little break! It is understandable you can get it by clicking on the :sloth: emoji below. You can send me !menu again to remove it.\n**BE ADVISED: THIS COMMAND CAN ONLY BE USED EVERY 24 HOURS**"
```

Comment:
```suggestion
    "It looks like you want a little break! It is understandable you can get it by clicking on the :sloth: emoji below. You can send me !menu again to remove it.\n**Be advised: You can only toggle your break role every 24 hours**"
```
No need to yell at our poor members ;) Also update the wording to clarify that it's not the menu command that goes on cooldown but toggling the break role.